### PR TITLE
Portfolio index page (Mobile skeleton)

### PR DESCRIPTION
**Pull request by Agetuni**
Add index page with with mobile skeleton desing
style the whole page 
Setup git ignore

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            Aleazar Yilma
+        </div>
+        <nav class="navigation">
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <main class="content">
+        <section class="title">
+            Aleazar Yilma
+
```
Comment (mwanawabangona) [inline index.html]: - [ ] :wave: `required` kindly add the name in and `h1` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        </section>
+        <section class="occupation">
+            Software Developer
+        </section>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required` please the software occupation in a tag as well `h1` or `h2`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            <a href="">
+                <img src="/images/icons/peace.png" alt="peace">
+            </a>
+            <a href="">
+                <img src="/images/icons/twitter.png" alt="peace">
+            </a>
+            <a href="">
+                <img src="/images/icons/m.png" alt="M">
+            </a>
+        </section>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required` Kindly add this in a `ul` and `li` see screenshot below 
Screenshot from 2022-06-29 13-54-35

- [ ] `required` please also fix the erros see screenshot below

your image
Screenshot from 2022-06-29 15-04-50

figma template
Screenshot from 2022-06-29 15-08-46

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @Agetuni  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional briefcase
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Agetuni , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**